File: App/Georentate/app/src/main/java/com/example/zilvinastomkevicius/georentate/Entities/Checkpoint.java
package com.example.zilvinastomkevicius.georentate.Entities;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Checkpoint entity
 */

public class Checkpoint {

    @JsonProperty("id")
    private int id;
    @JsonProperty("name")
    private String name;
    @JsonProperty("scan")
    private String scan;
    @JsonProperty("hint")
    private String hint;
    @JsonProperty("latitude")
    private double latitude;
    @JsonProperty("longitude")
    private double longitude;
    @JsonProperty("points")
    private float points;

    private int ExactDistance;
    private String ApproximateDistance;
    private String TravelTime;

    //Setters
    public void setId(int id) {
        this.id = id;
    }
    public void setName(String name) {
        this.name = name;
    }
    public void setScan(String scan) {
        this.scan = scan;
    }
    public void setHint(String hint) {
        this.hint = hint;
    }
    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }
    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }
    public void setPoints(float points) {
        this.points = points;
    }

    public void setExactDistance(int exactDistance) {
        this.ExactDistance = exactDistance;
    }
    public void setApproximateDistance(String approximateDistance) {
        this.ApproximateDistance = approximateDistance;
    }
    public void setTravelTime(String travelTime) {
        this.TravelTime = travelTime;
    }

    //Getters
    public int getId() {
        return id;
    }
    public String getName() {
        return name;
    }
    public String getScan() {
        return scan;
    }
    public String getHint() {
        return hint;
    }
    public double getLatitude() {
        return latitude;
    }
    public double getLongitude() {
        return longitude;
    }
    public float getPoints() {
        return points;
    }

    public int getExactDistance() {
        return ExactDistance;
    }
    public String getApproximateDistance() {
        return ApproximateDistance;
    }
    public String getTravelTime() {
        return TravelTime;
    }
}

File: API/GeorentateTrueAPI/ClassLibrary/BusinessEntities/Checkpoint.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ClassLibrary.BusinessEntities
{
    public class Checkpoint
    {
        public int id { get; set; }
        public string name { get; set; }
        public string scan { get; set; }
        public string hint { get; set; }
        public double latitude { get; set; }
        public double longitude { get; set; }
        public double points { get; set; }

        public Checkpoint(int id, string name, string scan, string hint, double latitude, double longitude, double points)
        {
            this.id = id;
            this.name = name;
            this.scan = scan;
            this.hint = hint;
            this.latitude = latitude;
            this.longitude = longitude;
            this.points = points;
        }

        public Checkpoint()
        {
        }
    }
}
